OpenRA.Game/Graphics/SpriteLoader.cs

```diff
@@ -21,6 +22,12 @@ public interface ISpriteLoader
 		bool TryParseSprite(Stream s, out ISpriteFrame[] frames);
 	}
 
+	public interface ISpriteLoader2 : ISpriteLoader
```

Comment:
Ok. I think I found solution. I create two classes. one can provide just tiles. will need SpriteFormats: IcnD2 for example. another will provide composed buildings. I name it MapD2.
I prefer to use the second. first will be available for use too. Will commit this update soon.